docs/dispatch/Error_Codes.md

```diff
@@ -14,7 +14,7 @@ This page outlines some of the common errors codes that may be encountered when
 | 2025 | Install Script Failed      | Restart Discord, attempt to uninstall/reinstall the game, ensure script is correct                                                             |
 | 2029 | Build Not Found            | Completely close and re-open Discord                                                                                                           |
 | 2051 | Panic!                     | Escalate in the dev server in #dispatch                                                                                                        |
```

Comment:
If you "fix" something at least fix it everywhere, aye? Unless there's no real interest in actually fixing anything, of course 😛